products/run-run-and-sweep-github-files/faq.md

```diff
@@ -4,6 +4,23 @@ description: Frequently asked questions about Runs
 
 # FAQ
 
+## How do I retain my run data?
```

Comment:
**How do I retain my run data?**
In your response, I see answers to two different questions
1. How do I download my run/experiment artifacts? 
2. How can I download data from an Interactive Session?

We should also get into the habit of giving an example of how to do something whenever we tell users to do it. 
1. download from jupyter hub (show screenshot of where)
2. mount the data locally with grid interactive mount (give example cli command)
3. scp (give an example command)
4. pause and restart (quick gif or screenshots)